hermes/utils/book_crossing_etl/bookcrossing.py

```diff
@@ -257,8 +257,8 @@ def parse_book_line(line):
 
     with\
         open(args.ratings, 'rb') as csvfile,\
-        open("book-crossing_implicit_ratings.json", 'w') as imp,\
-        open("book-crossing_explicit_ratings.json", 'w') as exp:
+        open("implicit_ratings.json", 'w') as imp,\
```

Comment:
Hey @tiffanyj41: Looks like you might have had a bad merge in your history, or a failed rebase? Your commit reverts some of mine.